gallifrey/core/src/main/java/gallifrey/core/SharedObject.java

```diff
@@ -115,6 +115,15 @@ public Object const_call(String FunctionName) {
         GenericFunction func = new GenericFunction(FunctionName);
         try {
             return getBackend().rmiOperation(this.key, func);
+        } catch (BackendRequiresFlushException b) {
+            getFrontend().static_read(b.key);
+            try {
+                return getBackend().rmiOperation(this.key, func, b.time);
+            } catch (RemoteException e) {
+                e.printStackTrace();
+                System.exit(99);
+                return null;
+            }
         } catch (RemoteException e) {
             e.printStackTrace();
             System.exit(22);
```

Comment:
How about we just replace this whole function body with `return const_call(FunctionName, new ArrayList<Object>())`